src/lib/elements/forms/inputTextarea.svelte

```diff
@@ -55,10 +67,22 @@
             class="input-text"
             bind:value
             bind:this={element}
-            on:invalid={handleInvalid} />
-        {#if maxlength}
-            <TextCounter max={maxlength} count={value?.length ?? 0} />
-        {/if}
+            on:invalid={handleInvalid}
+            style:--amount-of-buttons={required ? undefined : 0.25} />
+        <ul
+            class="buttons-list u-gap-8 u-cross-center u-position-absolute d u-inset-block-end-1 u-inset-inline-end-1 u-padding-block-8 u-padding-inline-12"
```

Comment:
Is it worth creating a "inputButtonContainer" component to avoid repeating all these classes?

Reply:
Maybeee, but if so we should probably do it in pink, right?

Reply:
Mmmmm maybe :thinking: